doc/qbk/02_6_allocators.qbk

```diff
@@ -23,19 +23,13 @@ library's polymorphic allocator but with the following additional features:
   the managed object to be extended.
 
 * An implementation can indicate to the library that deallocation
-  is not necessary, improving the performance of destructors.
+  is not necessary, allowing destructor calls to be elided in certain situations.
```

Comment:
"elided" yuck, use normal words! "allowing calls to destructors for elements of containers to be skipped in most cases."